orchagent/watermarkorch.cpp

```diff
@@ -209,6 +210,12 @@ void WatermarkOrch::doTask(NotificationConsumer &consumer)
         "SAI_BUFFER_POOL_STAT_WATERMARK_BYTES",
         gBufferOrch->getBufferPoolNameOidMap());
     }
+    else if (data == CLEAR_HEADROOM_POOL_REQUEST)
+    {
+        clearSingleWm(table,
+        "SAI_BUFFER_POOL_STAT_XOFF_ROOM_WATERMARK_BYTES",
+        gBufferOrch->getBufferPoolNameOidMap());
```

Comment:
This indentation looks strange. Did you use tab? Please indent one level (like other code in this file) or indent with '(' (preferred).

Reply:
No. I did not use tab. I just followed the indentation structure used in the rest of the else if blocks in this file. This code starts at the same column within the block as the ones above and below it when I check in 'vi'.

Reply:
Done. fixed all to align with brackets